ecl/mvna/v1/_proxy.py

```diff
@@ -361,7 +357,6 @@ def upload_certificate(self,
         body = {'type': certificate_type, 'file': certificate_file}
```

Comment:
```suggestion
        body = {'type': certificate_type, 'content': certificate_content}
```
パラメータ名が変更されているので修正をお願いします。